### PR TITLE
update to flat updatepayload structure

### PR DESCRIPTION
**Pull request by juanvallejo**
Updates Dockerfile to include a /manifests directory in image.

cc @deads2k @mfojtik @sttts

### CODE REVIEW

File: manifests/00_clusteropenshiftcontrollermanageroperator_01_config.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: openshiftcontrollermanageroperatorconfigs.openshiftcontrollermanager.operator.openshift.io
+spec:
+  scope: Cluster
+  group: openshiftcontrollermanager.operator.openshift.io
+  version: v1alpha1
+  names:
+    kind: KubeApiserverOperatorConfig
```
Comment (deads2k) [inline manifests/00_clusteropenshiftcontrollermanageroperator_01_config.crd.yaml]: typo

File: manifests/00_clusteropenshiftcontrollermanageroperator_02_config.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: openshiftcontrollermanager.operator.openshift.io/v1alpha1
+kind: KubeApiserverOperatorConfig
```
Comment (deads2k) [inline manifests/00_clusteropenshiftcontrollermanageroperator_02_config.yaml]: typo

**Comment by juanvallejo** [comment]
@deads2k thanks, review comments addressed

**Comment by juanvallejo** [comment]
@deads2k Added "categories" back to CRD.

**Pull request merged**